cybergis_compute_client/CyberGISCompute.py

```diff
@@ -362,6 +362,41 @@ def list_container(self, raw=False):
         else:
             print(MarkdownTable.render(data, headers))
 
+    def list_jupyter_host(self, raw=False):
+        """
+        Prints a list of jupyter hosts that the server supports
+
+        Args:
+            raw (bool): set to True if you want the raw output
+
+        Returns:
+            JSON: Raw output if raw=True otherwise its
+            printed or displayed directly into the interface
+        """
+        try:
```

Comment:
Thank you for adding the try/catch block for backward compatibility! We probably would want to get more explicit in the except (i.e. catch specific exceptions), but this likely wont be used very much and it works for now. I tested it out and it works fine.